CHANGES.rst

```diff
@@ -27,6 +27,13 @@ calwebb_detector1
 - Added regression test for ``calwebb_detector1`` pipeline which now
   includes ``undersampling_correction``. [#7509]
 
+calwebb_spec2
```

Comment:
It isn't the calwebb_spec2 pipeline module that's being updated here, it's the individual steps (resample_spec and extract_1d)

Reply:
Fixed